sample-compose-fragments/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Do we need these? Ideally we'd be using all the new colours from the compose library, rather than any hardcoded in the xml in the app.